actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1668,7 +1668,7 @@ def initialize(object_name, object, template, options)
 
         convert_to_legacy_options(@options)
 
-        if @object_name.to_s.match(/\[\]$/)
+        if @object_name.to_s.match(/\[\]$/) # rubocop:disable Performance/EndWith
```

Comment:
Why use `rubocop:disable`?

Reply:
delete_suffix cannot be used - matchdata is used in the next line (`Regexp.last_match.pre_match`)

Reply:
Use `if foo.end_with("[]"); foo.delete_suffix("[]")` as a faster alternative.

Reply:
ok 👍 
Updated, and in other places. CI failures seems not related to this PR.